### ISSUE TITLE
backup hangs forever without no output

### ISSUE THREAD

**Issue by jarmo** [duplicati/duplicati#2351]
I have:
- [x ] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.1.50_canary_2017-02-28
**Operating System:** Ubuntu 14.04
**Backend:** webdav

### Bug description
Backup failed and so did repair. Deleted local database and started backup again. After few hours process is still running, but nothing seems to happen. In profile log, last line is:
```
2017-03-03 11:02:14Z - Warning: Extra unknown file: duplicati-i5086befffe154c2390981dcf5e2bf191.dindex.zip.aes
```

This was over 3h ago.

**Comment by kenkendk** [comment]
So it is the "repair" that hangs, not the backup itself?

**Comment by jarmo** [comment]
No, backup itself after deleting local database, but leaving remote (physical disk with backup directory) intact.

**Comment by kenkendk** [comment]
Ok, so the repair completed, but now the backup cannot run.

Have you tried stopping it and starting again (not a real solution, just to check if it can be easily reproduced)?

**Comment by kenkendk** [comment]
Maybe it is related to #2336

**Comment by jarmo** [comment]
No, repair failed and that's why I deleted local database, because before that backup failed. So I deleted local database to rebuild database with performing yet another incremental backup, which hanged "forever". It was reproducible at the time by killing process and trying again.

However, I ended up deleting remote backup completely and doing a new full backup, which worked so I cannot reproduce it anymore at this time. Just created this issue if problem should happen for anyone else or for me again. Until that there's no extra information to provide by me.

Maybe close it for now and I will reopen it again if there's any extra information to provide?

**Comment by ts678** [comment]
> Maybe close it for now and I will reopen it again if there's any extra information to provide?

This sounds like a good idea because the problem was removed by the full backup, and there's not much information currently.
You can do a close, or if no more information or action occurs, someone can manually do the close while cleaning up old issues.
